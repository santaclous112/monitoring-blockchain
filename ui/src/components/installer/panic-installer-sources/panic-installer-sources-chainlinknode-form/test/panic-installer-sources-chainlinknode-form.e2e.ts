import {newSpecPage} from '@stencil/core/testing';
import {
  PanicInstallerSourcesChainlinkNodeForm
} from "../panic-installer-sources-chainlinknode-form";

describe('panic-installer-sources-chainlinknode-form', () => {
  it('renders', async () => {
    const page = await newSpecPage(
        {
          components: [PanicInstallerSourcesChainlinkNodeForm],
          html: '<panic-installer-sources-chainlinknode-form></panic-installer-sources-chainlinknode-form>',
        }
    );

    const element = page.body.querySelector('panic-installer-sources-chainlinknode-form');
    expect(element).toMatchSnapshot();
  });
});
