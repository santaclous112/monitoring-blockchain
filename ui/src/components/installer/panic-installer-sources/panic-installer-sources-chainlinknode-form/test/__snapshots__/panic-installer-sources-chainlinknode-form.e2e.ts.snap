// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-installer-sources-chainlinknode-form renders 1`] = `
<panic-installer-sources-chainlinknode-form>
  <svc-input label="Node Name" lines="inset" name="name" placeholder="chainlink_node_1(IP)" required=""></svc-input>
  <div class="panic-installer-sources-form__help-text">
    <span>
      This unique identifier will be used to identify your node.
    </span>
  </div>
  <div class="panic-installer-sources-form__row-container">
    <div class="panic-installer-sources-form__input_test_box">
      <svc-multiple-input addeventname="AddedPrometheusURL" label="Prometheus URLs" name="nodePrometheusUrls" outline="" placeholder="https://IP:6688/metrics [Press Enter after each URL]" removeeventname="RemovedPrometheusURL" type="url"></svc-multiple-input>
      <svc-toggle checked="" label="Monitor" lines="none" name="monitorPrometheus" value=""></svc-toggle>
      <panic-installer-test-button-multiple-sources service="prometheus"></panic-installer-test-button-multiple-sources>
    </div>
  </div>
  <div class="panic-installer-sources-form__help-text">
    <span>
      These are the prometheus URLs of your node.
    </span>
  </div>
  <svc-toggle checked="" helptext="Whether to monitor this configured node." label="Monitor Node" lines="inset" name="monitorNode" value=""></svc-toggle>
</panic-installer-sources-chainlinknode-form>
`;
